ui/epicSkills/epicSkills.pug

```diff
@@ -0,0 +1,51 @@
+// skill section 
```

Comment:
I recommend reviewing by commit, and also by ignoring whitespace:
https://github.com/epic-power-rpg/roll20/pull/58/commits/abae571e20d6b919777a47bf9314ac59f0db868d?diff=unified&w=1

I scanned through everything and didn't see any breaking issues.